maps/torch/items/summarydocuments.dm

```diff
@@ -1,393 +1,375 @@
 
 /obj/item/document/generic/scgr/deskfires
 	name = "important notice"
-	desc = "\"Confidential\" notice issued to Sol Government Representatives, informing them of an office fire caused by improper cigarette ash management."
+	desc = "A \"confidential\" notice issued to Sol Government Representatives, informing them of an office fire caused by improper cigarette ash management."
 
 /obj/item/document/generic/scgr/boringreports
 	name = "Torch project status update #132"
-	desc = "\"Confidential"\" report containing nothing you'd like to read, featuring updates that you knew about three weeks ago."
+	desc = "A \"confidential\" report containing nothing you'd like to read, featuring updates that you knew about three weeks ago."
 
 /obj/item/document/generic/scgr/documentdamage
 	name = "important notice"
-	desc = "\"Confidential\" notice issued to Sol Government Representatives, informing them many documents were damaged last week due to \"inappropriate sleep management\". It urges all diplomatic staff to self-monitor their sleeping schedule and to regulate their caffeine intake."
+	desc = "A \"confidential\" notice issued to Sol Government Representatives, informing them that many documents were damaged last week due to \"inappropriate sleep management\". It urges all diplomatic staff to self-monitor their sleeping schedule and to regulate their caffeine intake."
 
 /obj/item/documents/generic/scgr/documentdisposal
 	name = "proper document disposal"
-	desc = "\"Internal Use ONLY\" documents, showing several pictures of reconstructed shredded documents. It highlights that shredding is not acceptable for any documents that are above Confidental, going on to recommend incineration as a primary method."
+	desc = "Some \"internal use only\" documents, showing several pictures of reconstructed shredded documents. It highlights that shredding is not acceptable for any documents that are above Confidental, going on to recommend incineration as a primary method."
 
 /obj/item/documents/generic/scgr/yourjob
 	name = "SCG and you"
-	desc = "\"Confidential\" briefing documents, detailing the job of the SCGR in brief. It highlights they are a Civilian, who functions as a general purpose civil servant onboard the SEV Torch, while also advising the Commanding Officer, assisting with any diplomacy that may occur, facilitating the navigation of bureacracy, and their specific training in regards to the SEV Torch's mission."
+	desc = "A \"confidential\" briefing documents, detailing the job of the SCGR in brief. It highlights they are a civilian, first and foremost, who functions as a general-purpose civil servant aboard the SEV Torch. They also should advise the Commanding Officer, assisting with bureaucratic affairs where required, and spearheading diplomatic negotations."
```

Comment:
Some confidential documents, not A confidential documents.

Reply:
Fixed.